### PR TITLE
Add blurb on interactive plot features

### PR DESCRIPTION
**Pull request by e-koch**
Addresses @snlongmore 's issue to explain the interactive plot #132

### CODE REVIEW

**Comment by e-koch** [comment]
screen shot 2015-06-18 at 10 34 23 am

**Comment by e-koch** [comment]
Added a line to explain how to reset the plot

**Comment by jmdkastro** [comment]
Great! Perhaps also add a line about the home button?

**Comment by e-koch** [comment]
Just did! Here's an update view,

screen shot 2015-06-18 at 10 38 31 am

**Comment by e-koch** [comment]
I think the text covers all of the features now. Anyone have any other suggestions?

File: templates/show_plot.html [hunk 1]
```diff
@@ -28,6 +28,20 @@ <h1 class="title">CAMELOT Query output</h1>
...
      {% endif %}
+     <div class="row" align="left">
+        <div class="col-md-10">
+            <div>
+            <b>Plot Features:</b>
+            </div>
+            <ul style="list-style-type:none">
+              <li> <b>Zoom</b> - Press the magnifying glass on the bottom left of the plot. </li>
+              <li> <b>Pan</b> - Press the arrow button on the bottom left of the plot. In pan mode, you can zoom by using the mouse wheel. </li>
+              <li> <b>Hover</b> - Mousing over a point in the plot shows it's physical properties as well as the metadata for the dataset it belongs to. </li>
```
Comment (keflavich) [inline templates/show_plot.html]: it's -> its

File: templates/show_plot.html [hunk 2]
```diff
@@ -28,6 +28,20 @@ <h1 class="title">CAMELOT Query output</h1>
...
     <div id="link_pull_uploads">Submitted file pull request: <a href="{{link_pull_uploads}}">{{link_pull_uploads}}</a></div>
      {% endif %}
+     <div class="row" align="left">
+        <div class="col-md-10">
+            <div>
+            <b>Plot Features:</b>
+            </div>
+            <ul style="list-style-type:none">
+              <li> <b>Zoom</b> - Press the magnifying glass on the bottom left of the plot. </li>
+              <li> <b>Pan</b> - Press the arrow button on the bottom left of the plot. In pan mode, you can zoom by using the mouse wheel. </li>
```
Comment (keflavich) [inline templates/show_plot.html]: arrow -> crossed arrows?

**Comment by keflavich** [comment]
:+1:

**Comment by e-koch** [comment]
Fixed! Merging.

**Pull request merged**